### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ecenursahin**
Added some explanation about the SMS and Email feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,13 +83,101 @@ https://medium.com/datadriveninvestor/block-chain-using-javas-web3j-and-ethereum
...
 
+### SMS & Email Functionalites:
 
+For this project we decided to use Twilio SMS hosting service API and JavaMail API. 
 
+These features allow:
 
+#### **Sending SMS notifications when a user:**
 
+Receives 2 factor authentication passcode, makes withdraw, makes Deposit, creates a new bank account, removes an existing bank account, makes a loan request, gets their loan request approved or rejected, and makes a loan payment.
```
Comment (AllenNinous) [inline README.md:94]: Deposit with a lowercase 'd'?

File: README.md [hunk 2]
```diff
@@ -83,13 +83,101 @@ https://medium.com/datadriveninvestor/block-chain-using-javas-web3j-and-ethereum
...
 
+#### **Sending Email notifications when a user:**
 
+Wants to receive the recent transactions statements
 
+Optionally a user also receives additional notifications when a user makes withdraw, makes Deposit, creates a new bank account, removes an existing bank account, makes a loan request, and makes a loan payment. (The console asks for a user if they want to receive an email copy of the transaction notice in addition to the SMS notification)
```
Comment (AllenNinous) [inline README.md:100]: Same as the comment above regarding "Deposit"

File: README.md [hunk 3]
```diff
@@ -83,13 +83,101 @@ https://medium.com/datadriveninvestor/block-chain-using-javas-web3j-and-ethereum
...
 
+Because we are using secret login IDs and passwords that we do not want to share publicly, we are using a secret directory to store those credential variables.
+For grading purposes we will provide the credentials to our class instructors only.
+
+
+If you are not an Instructor, all you need to do is:
+
+1-Create a Twilio Account - trial version is fine- and
+
+2-A gmail account.
```
Comment (AllenNinous) [inline README.md:111]: Does "gmail" need to be written as "Gmail" with an uppercase 'G'

File: README.md [hunk 4]
```diff
@@ -83,13 +83,101 @@ https://medium.com/datadriveninvestor/block-chain-using-javas-web3j-and-ethereum
...
+### Using the JavaMail API To Send Emails Using a GMail Account.
+For this project we are using JavaMail API. In our code we are using gmail SMTP setup. 
+
+The sender email for the bank system is the team gmail. The following login credentials are also stored in the secret folder shared with the instructors since we login to our email using the API and send our message to any email receiver.(Used in Email.java)
+
+SENDER_EMAIL 
+
+SENDER_PASSWORD 
+
+Here is an exaple email sent. The sender needs to be a Gmail with our SMTP setup for the gmail, however, the receiver could be using any emai service.
```
Comment (AllenNinous) [inline README.md:175]: "email" missing an 'l'

**Review by AllenNinous — CHANGES_REQUESTED** [review]
Hey Ece. This is very nicely and neatly done. I've gone through it quickly and I understood everything in there. 
I left 4 comments highlighting typos(?) and I'm happy to approve once they're addressed. Thank you so much for doing this.

**Review by AllenNinous — APPROVED** [review]
Looks great @Ecenursahin! Approved. Thank you so much.

File: README.md (hunk 3 above)
Comment (Ecenursahin) [reply README.md]: Made changes to all gmail s to write as Gmail for consistency, thanks for catching this! :)

File: README.md (hunk 4 above)
Comment (Ecenursahin) [reply README.md]: Made the change! Thank you

**Comment by Ecenursahin** [comment]
Thank you so much for the review Allen!

**Pull request merged**